### PR TITLE
Add 10y localhost-recovery serving cert

### PR DESCRIPTION
**Pull request by sttts**
This will be used by the cert recovery sidecar.

### CODE REVIEW

**Comment by tnozicka** [comment]
/hold
Should we wire it for the cert-syncer as well in this PR? We want to use it there anyways and using it will make sure it is tested

File: pkg/operator/certrotationcontroller/certrotationcontroller.go [hunk 1]
```diff
@@ -322,6 +322,47 @@ func NewCertRotationController(
...
+			Name:          "localhost-recovery-serving-ca",
+			Informer:      kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().ConfigMaps(),
+			Lister:        kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().ConfigMaps().Lister(),
+			Client:        kubeClient.CoreV1(),
+			EventRecorder: eventRecorder,
+		},
+		certrotation.TargetRotation{
+			Namespace: operatorclient.TargetNamespace,
+			Name:      "localhost-recovery-serving-certkey",
+			Validity:  10 * 365 * rotationDay,
```
Comment (soltysh) [inline pkg/operator/certrotationcontroller/certrotationcontroller.go]: I think both should be `defaultRotationDay` we don't want to tweak this value ever.

Comment (sttts) [reply pkg/operator/certrotationcontroller/certrotationcontroller.go]: fixed

**Comment by sttts** [comment]
Unrelated flakes.

/retest

**Comment by sttts** [comment]
Unrelated flakes?

/retest

**Comment by sttts** [comment]
Unrelated flakes?

/retest

**Comment by tnozicka** [comment]
/hold
the operator test fails consistently, needs to be addressed

**Comment by tnozicka** [comment]
/hold cancel
e2e-aws-operator that was broken now succeeded

**Pull request merged**